### PR TITLE
SW-2629 Remove id from on change handlers

### PR DESCRIPTION
**Pull request by constanzauanini**
This change is needed so html id can be set independently of the data model. We can then change the value of the data model using the necessary id, separately of the html id

### CODE REVIEW

**Review by karthikbtf — COMMENTED** [review]
Thanks for doing this @constanzauanini 
I think you need to update the stories as well.
Maybe run storybook just to test.

**Review by karthikbtf — APPROVED** [review]

**Comment by karthikbtf** [comment]
oops, i've been merging the other PRs,, i will hold off until this merged

**Comment by constanzauanini** [comment]
> oops, i've been merging the other PRs,, i will hold off until this merged

Merged! Thanks

**Pull request merged**
